dashboard/app/models/plc/user_course_enrollment.rb

```diff
@@ -97,18 +96,8 @@ def summarize
   private
 
   def create_authorized_teacher_user_permission
-    unless user.verified_teacher?
+    if user.teacher? && !user.verified_teacher?
       user.permission = UserPermission::AUTHORIZED_TEACHER
     end
   end
-
-  def delete_authorized_teacher_user_permission
```

Comment:
It does not make sense to take away a teachers verified status once they have received it